Slider/Assets/Scripts/Audio/AudioManager.cs

```diff
@@ -301,4 +320,76 @@ public static void SetSFXPitch(float value)
             s.source.pitch = s.pitch * value;
         }
     }
+
+    public static void EnqueueModifier(AudioModifier.ModifierType m)
+    {
+        if (_instance.modifiers.ContainsKey(m))
+            _instance.EnqueueModifier(_instance.modifiers[m]);
+        else
+            Debug.LogError($"Trying to access non-materialized modifier {m}");
+    }
+
+    private void EnqueueModifier(AudioModifier m)
+    {
+        foreach (var adj in m.adjustments)
+        {
+            var name = adj.parameter;
```

Comment:
nit: don't use `var` for built-in types, like strings and floats -- makes more readable imo

if its something annoying like `ParticleSystem.ParticleModule.BananaSpriteModifier` or whatever then its fine